apps/dot-voting/app/Decisions.js

```diff
@@ -107,11 +110,19 @@ const Decisions = ({ decorateVote }) => {
   const {
     votes,
     voteTime,
+    tokenAddress = '',
   } = appState
 
   const { layoutName } = useLayout()
   const theme = useTheme()
 
+  const tokenAbi = [].concat(tokenBalanceOfAbi, tokenDecimalsAbi, tokenSymbolAbi)
```

Comment:
This is a constant that is not reliant on component state/props. Let's move it outside of the component definition, right under the `import tokenDecimalsAbi from './abi/token-decimals.json'` part. That is, if we even need it anymore, given that it seems we only need `tokenDecimalsAbi`.

Reply:
Done.